runtime/regexp.cpp

```diff
@@ -751,7 +751,7 @@ Optional<int64_t> regexp::match(const string &subject, mixed &matches, bool all_
         matches[i].push_back(match_str);
       }
     } else {
-      int64_t size = fix_php_bugs ? subpatterns_count : count;
+      int64_t size = count;
```

Comment:
May be we can drop `size` and use `count` directly?
Same for diff below.

Reply:
Fixed.